src/resources/js/views/events-bar.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events = tribe.events || {};
+tribe.events.views = tribe.events.views || {};
+
+/**
+ * Configures Events Bar Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events.views.eventsBar = {};
+
+/**
+ * Initializes in a Strict env the code that manages the Event Views
+ *
+ * @since TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} obj tribe.events.views.manager
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	var $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		input: '.tribe-common-form-control-text__input',
```

Comment:
we should try and use `[data-js]` as much as possible, except where we're actually adding classes.

Reply:
Where would you suggest it?

Reply:
on the element, we can add `data-js` attributes and selectors used in JS can be `[data-js="the-selector"]`. if you're not sure, let's hop on a call and walk through it.

Reply:
Yep. I was asking in this particular case if you were suggesting to do that in a particular thing.